### PR TITLE
TASK: Polish reference section of documentation

### PR DESCRIPTION
**Pull request by mficzel**
The typoscript view-helper is added to the autogenereated neos-reference.  
Additionally the section names in the reference are made a little more consistent.

### CODE REVIEW

**Comment by gerhard-boden** [comment]
:+1: for the text changes, can't really tell why the tests fail. Any idea?

File: TYPO3.Neos/Configuration/Settings.yaml [hunk 1]
```diff
@@ -599,6 +599,18 @@ TYPO3:
             namespaces:
               f: 'TYPO3\Fluid\ViewHelpers'
               neos: 'TYPO3\Neos\ViewHelpers'
+      'Neos:TypoScriptViewHelpers':
```
Comment (gerhard-boden) [inline TYPO3.Neos/Configuration/Settings.yaml]: should this be part of the change?

Comment (mficzel) [inline TYPO3.Neos/Configuration/Settings.yaml]: Yes this is intended. Currently the view ts:render view helper is simply missing in the docs.

**Comment by mficzel** [comment]
I also have no clue why the travis-test failed. I altered no code at all so this has to be a test-infrastructure issue. Can anyone trigger a second run of the failed test?

**Comment by johannessteu** [comment]
Closed / reopened to trigger tests

**Comment by gerhard-boden** [comment]
This is where the tests seem to fail
```
Cloning into 'neos/neos-development-collection'...
warning: Could not find remote branch task/polishReferenceSectionOfDocs to clone.
fatal: Remote branch task/polishReferenceSectionOfDocs not found in upstream origin
Unexpected end of command stream
The command "eval git clone --depth=50 --branch=task/polishReferenceSectionOfDocs https://github.com/neos/neos-development-collection.git neos/neos-development-collection" failed. Retrying, 2 of 3.
```

File: TYPO3.Neos/Configuration/Settings.yaml (hunk 1 above)
Comment (gerhard-boden) [inline TYPO3.Neos/Configuration/Settings.yaml]: okay, thanks for the information!
I have no deeper knowledege how docs are generated, but the `TypoScipt.rst` file does not show up here:
https://github.com/neos/neos-development-collection/tree/master/TYPO3.Neos/Documentation/References/ViewHelpers
That's why i was wondering

**Pull request merged**